alerting/src/main/resources/com/amazon/opendistroforelasticsearch/alerting/alerts/alert_mapping.json

```diff
@@ -34,6 +34,43 @@
         }
       }
     },
+    "monitor_user": {
```

Comment:
As discussed offline, since `"schema_version"` has been incremented after the last ODFE release already, we don't necessarily need to increment it again for this change.